angular-ngrx-scss/src/app/repository/components/repo-header/repo-info/repo-info.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-repo-info',
+  templateUrl: './repo-info.component.html',
+  styleUrls: ['./repo-info.component.scss'],
+})
+export class RepositoryInfoComponent {
+  @Input() watch = 675;
```

Comment:
suggestion: We should also have these values available, but I'm guessing these are simply default values, correct? If so, I think it'd be best (once you're done testing things) to set these to default to 0.